nixos/doc/manual/installation/installing.chapter.md

```diff
@@ -1,30 +1,127 @@
+
+3.   Press <kbd>F12</kbd> (or the appropriate key) and select the install drive,
+usually "USB-HDD" or something containing the word "USB", but the wording may vary.
+If your computer support BIOS and UEFI boot, choose the UEFI option.
+If you choose the incorrect drive, your computer will likely continue to boot
+as normal. Just restart your computer and pick a different drive in that menu.
+
+4.   Shortly after selecting the appropriate boot drive, you should be presented
+with a menu with different Installer options. Leave the default and wait
+(or press <kbd>Enter</kbd> to speed up).
+
+5.   The graphical images will start the desktop environment and the graphical
+installer, which can take some time. The minimal images will just have
+a command line.
+
+## Graphical Installation {#sec-installation-graphical}
+
+The graphical installer is recommended for desktop users and will guide you
+through the installation.
+
+1.   In the "Welcome" screen, you can select the language of the Installer and
+     the installed system.
+
+     ::: {.tip}
+     Leaving the language as "American English" will help to be able to copy
+     and paste error messages in a search engine or issue report.
+     :::
+
+2.   Next you should choose your location to have the timezone set correctly.
+     You can actually click on the map!
+
+     ::: {.note}
+     The installer will use an online service to guess your location based on
+     your public IP address.
+     :::
+
+3.   Then you can select the keyboard layout. The default keyboard model should
+     work well with most desktop keyboards. If you have a special keyboard or
+     notebook, your model might be in the list. Select the language you are most
+     comfortable typing in.
+
+     ::: {.tip}
+     There are special keyboard layouts that are optimized to be ergonomic like
+     [NEO](https://www.neo-layout.org/),
+     [Dvorak](https://en.wikipedia.org/wiki/Dvorak_keyboard_layout) and
+     [Colemak](https://colemak.com/), in opposition to the classic QWERTY layout
+     which comes from typewriters and has not changed much since 1873.
+     You might want to check them out.
+     :::
 
-The installation media can be burned to a CD, or now more commonly,
-"burned" to a USB drive (see [](#sec-booting-from-usb)).
+4.   On the "Users" screen, you have to type in your name, login name and
+     password. You can also enable an option to automatically login to the
+     desktop.
 
-The installation media contains a basic NixOS installation. When it's
-finished booting, it should have detected most of your hardware.
+5.   Then you have the option to choose a desktop environment. If you want to
+     create a custom setup with a window manager, you can select "No desktop".
 
-The NixOS manual is available by running `nixos-help`.
+     ::: {.tip}
+     If you don't have a favorite desktop and don't know which to choose, you
+     can stick to either GNOME or Plasma. They are quiet different from their
```

Comment:
```suggestion
     can stick to either GNOME or Plasma. They are quite different from their
```

Quiet is silent.

```suggestion
     can stick to either GNOME or Plasma. They are quite different in their
```

I don't think "different from their design" is proper english, but I'm also an english-as-a-second-language person...

Reply:
right. "in their design" sounds more correct

i just use "They have a quite different design" now